adoc/deployment-preparation.adoc

```diff
@@ -116,7 +116,8 @@ during the cluster bootstrap. To simplify configuration you can reserve the IPs
 needed for the cluster nodes and pre-configure these in the load balancer.
 
 The load balancer needs access to port `6443` on the `apiserver` (all master nodes)
-in the cluster.
+in the cluster. It also needs access to Gangway port `32001` and Dex port `32002` 
+on all worker nodes in the cluster for RBAC authentication.
```

Comment:
Reference [v3](https://www.suse.com/documentation/suse-caasp-3/book_caasp_deployment/data/sec_deploy_requirements_network.html), both masters and workers expose ports.

![image](https://user-images.githubusercontent.com/49380831/60854419-ccd3ae00-a232-11e9-99de-e2a428f71d2b.png)

Reply:
Fixed, thanks :)